frappe/model/document.py

```diff
@@ -763,10 +763,11 @@ def check_if_latest(self):
 				self.check_docstatus_transition(tmp.docstatus)
 
 			if conflict:
+				# fmt: off
+				msg = _("Error: Document {2} has been modified after you have opened it ({0}, {1}). Please refresh to get the latest document")
```

Comment:
A bit conflicted on this, given this change invalidates the translations that currently exist in 66 languages. But the change makes the error more specific which is great.

```suggestion
				msg = _("Error: Document {0} has been modified after you have opened it ({1}, {2}).").format(self, modified, self.modified) + " " +
				+ _("Please refresh to get the latest document.")
```

Fixed numbering + translations will work/be retained with minimal change ^

Reply:
I understand that we'll lose those translations for the time being. But this change was added so that the error message is more specific which you have acknowledged.
Also, I am not in favor of breaking a sentence for translations. Every language is different and it's not guaranteed that separately translated strings will form a meaningful sentence.